### PR TITLE
Docs: Getting Started

### PR DESCRIPTION
**Pull request by johnyob**
# Context

**Related Tasks**: [Guide doc: Running sandbox](https://app.asana.com/0/1205170694555856/1205457666696801/f), [Guide doc: Contracts](https://app.asana.com/0/1205170694555856/1205457666696803/f)

# Description

This PR is the sequel to my comment https://github.com/trilitech/jstz/pull/31#issuecomment-1743045018. 

# Manually testing the PR

```sh
npm run docs:dev
```

### CODE REVIEW

**Review by sam-finch-tezos — APPROVED** [review]

**Comment by johnyob** [comment]
Will wait for additional approval from @alanmarkoTrilitech or @asutosh-mourya before merging

File: docs/first_steps.md
Comment (sam-finch-tezos) [inline docs/first_steps.md:101]: Is it worth doing some market research on the name _smart function_?

**Review by sam-finch-tezos — APPROVED** [review]
Approved, but have raised a few points you may or may not wish to address.

**Review by asutosh-mourya — APPROVED** [review]
Overall looks good. Sam raised couple of good questions which I think we should discuss (especially name smart function), however,  we can proceed with this as is for now and iterate over as we discuss and decide.

File: docs/index.md
Comment (johnyob) [reply docs/index.md:11]: I'd prefer to keep a pronunciation. Namely to avoid a mix of pronunciations + to explain any law-related puns we might make later on

File: docs/installation.md
Comment (johnyob) [reply docs/installation.md:81]: Correct 👍 . Fixing now

File: docs/installation.md
Comment (johnyob) [reply docs/installation.md:50]: I will leave in for now. I will update once I've confirmed it works on a standard octez distribution + modified the nix flake accordingly (in a separate PR)

File: docs/installation.md
Comment (johnyob) [reply docs/installation.md:81]: Actually the original file is correct, but running the sandbox needs a `make build-installer` command

**Pull request merged**
